### PR TITLE
Add `preview_image` and `preview_image_link` to the list of smart fields for resolveuid and link integrity

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Review by ericof — APPROVED** [review]
LGTM

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Review by tisto — APPROVED** [review]

**Review by davisagli — COMMENTED** [review]
@sneridagh @tisto Please don't release this, it doesn't look right to me.

File: src/plone/restapi/blocks_linkintegrity.py [hunk 1]
```diff
@@ -111,7 +110,7 @@ def __call__(self, block):
         Returns a list of internal links
         """
         links = []
-        for field in ["url", "href"]:
+        for field in ["url", "href", "preview_image", "preview_image_link"]:
```
[the remark this answers is not in the document]
Comment (davisagli) [reply src/plone/restapi/blocks_linkintegrity.py:113]: Sorry, I was reading it wrong, this is the generic blocks retriever. It makes sense for preview_image, which is used in the teaser block. It doesn't make sense for preview_image_link, which is a relation field not in blocks. I don't think we need to do anything special to make linkintegrity work for a relation field.

**Pull request merged**